README.md

```diff
@@ -129,7 +133,7 @@ Some examples include:
 - `fabrikor-3dprinter-nrf52-firmware`
 - `irnas-guidelines-docs`
 
-## Versioning and tagging 🏷️
+## Versioning 1️⃣0️⃣0️⃣
 
 The versioning scheme consists of 3 numbers separated by dots:
 ```
```

Comment:
here you have version without `v` and it is a bit confusing that you are mentioning versioning here and later on in releases. I understand this is meant for inside firmware, but it should be better structured.

Reply:
I understand now what you mean, it is unclear and needlessly different. I agree that it should be the same.

So I see two options:
* Add `v` in `Versioning` section
* Remove `v` from branch naming, release naming, tagging, binary naming(personally I would keep it there, cause you can also have hv version part, but it can be done), this would also mean that ci scripts need to change.

Reply:
yes, do it like that. That makes sense.